arch/xtensa/core/atomic.S

```diff
@@ -393,13 +393,14 @@ atomic_cas:
 	ENTRY(48)
 .L_LoopCas:
```

Comment:
This label looks like an orphan now?  Remove?

Reply:
Removed.